### ISSUE TITLE
Multi line descriptions of polls are not properly displayed.

### ISSUE THREAD

**Issue by makoehr** [nextcloud/polls#274]
### Steps to reproduce
 Create a new poll and enter a multi line description (with line breaks)

### Expected behaviour
The layout of the description that is visiable in the poll should be the same as with the edit window of the description.

### Actual behaviour
Instead of getting proper line breaks, the html tags <br/> <br> are displayed inline. No line break is visible!

### Server configuration

**PHP version:**
7.0.25

**Nextcloud version:** (see Nextcloud admin page)
12.0.4

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
No errors have been found.

**Comment by dartcafe** [comment]
Fixed in next version (0.8.1) see: https://github.com/nextcloud/polls/pull/260#issuecomment-349417659

**Comment by Mannshoch** [comment]
I use 0.8.1 at moment. How do I get multi line description work? Simple Line break do not work 
Also `<br />` do not work.
